ydb/src/lib.rs

```diff
@@ -143,8 +143,9 @@ pub use waiter::Waiter;
 // full enum pub types
 pub use crate::{
     credentials::{
-        CommandLineYcToken, GCEMetadata, ServiceAccountCredentials, StaticCredentialsAuth,
-        StaticToken, YandexMetadata,
+        AccessTokenCredentials, AnonymousCredentials, CommandLineCredentials, CommandLineYcToken,
+        FromEnvCredentials, GCEMetadata, MetadataUrlCredentials, ServiceAccountCredentials,
```

Comment:
What about list old value types separately and mark them as deprecated?

Reply:
Good idea!